docs/smart-contracts/tokens/wayfinding.mdx

```diff
@@ -0,0 +1,66 @@
+
+However:
+
+### TL;DR
+
+If possible, we recommend issuing a Stellar asset and (if needed) using the SAC to interact with that asset in smart contracts. More on why below.
+
+## Issuing assets on Stellar
+
+Stellar is first-class in asset tokenization — issuing an asset can be done in [just one single transaction](https://developers.stellar.org/docs).
+
+Stellar’s transactions are fast and cost-effective, making the network great for remittances and micropayments. It also has built-in features for compliance, asset management, and auditing. If you are just looking to perform simple transfers, issuing assets on Stellar has all the needed capabilities.
+
+Stellar assets:
+
+Are already widely supported in the Stellar ecosystem and other ecosystems (such as Stellar USDC on Coinbase). Benefit from [anchors](../../learn/encyclopedia/anchors.mdx), the bridges between the Stellar network and traditional financial systems. Explore the global [Stellar anchor directory](https://anchors.stellar.org/) for further details. Give the issuer granular control over asset management with features that allow the issuer to [publish asset information](../../issuing-assets/publishing-asset-info.mdx), [ensure compliance](../../issuing-assets/anatomy-of-an-asset.mdx#compliance), and [regulate asset supply](../../issuing-assets/control-asset-access.mdx#limiting-the-supply-of-an-asset).
+
+Assets issued on the Stellar network are accessible to smart contracts with the use of that asset’s Stellar Asset Contract (SAC).
+
+### Stellar Asset Contract
+
+The Stellar Asset Contract (SAC) is compiled into the protocol layer and allows smart contracts to interact with assets issued on Stellar. Every Stellar asset has reserved a SAC on the network that can be deployed by anyone who wants to interact with the asset from a contract. The SAC has access to all account (for XLM) and trustline (for all other assets) balances as well as smart contract token balances.
+
+Read more about the SAC [here](./stellar-asset-contract.mdx).
+
+Learn how to deploy a Stellar Asset Contract for an asset in [this How-To Guide](../../smart-contracts/guides/cli/deploy-stellar-asset-contract.mdx).
+
+**Benefits of the SAC:**
+
+- Compatibility: Stellar ecosystem products (such as Stellar wallets) and other ecosystem products (such as exchanges) already know how to interact with Stellar assets, making the SAC more interoperable.
+- Cost and resource efficiency: the SAC is built into the protocol-level code that already runs the network. So, there is no need to spin up a Wasm contract’s VM to invoke a function. Each function within the SAC will be more resource-efficient than its custom-coded counterpart.
+- Less work: you don’t have to write an entirely new contract. A Stellar asset’s SAC already exists on the network and just needs to be deployed to be used.
```

Comment:
We should add:
- Customization: Stellar assets have the following customizations: x, y, z, admin addresses can be contracts, etc.

Reply:
I added some things here. Mostly related to Stellar assets, not the SAC. LMK what you think!